plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java

```diff
@@ -47,21 +47,27 @@
     private final GenericContainer<?> master;
     private final List<GenericContainer<?>> tServers;
 
+    public TestingKuduServer()
+    {
+        // This version should match the kudu client version
```

Comment:
I don't think this comment is true? We can obviously mix older clients with newer servers or vice versa for most of the versions.

Reply:
I put this comment because other tests use the default `TestingKuduServer()` constructor. Only the smoke tests specifically specify the version of the kudu server to use. 

When we upgrade the client version, we presumably want to also upgrade all the [non-smoke tests] to use a newer version of the kudu server.